regression-test/suites/nereids_p0/expression/fold_constant/fold_constant_string_arithmatic.groovy

```diff
@@ -448,7 +564,65 @@ suite("fold_constant_string_arithmatic") {
     testFoldConst("select rtrim(cast('  Hello World  ' as string))")
     testFoldConst("select rtrim('Hello')")
     testFoldConst("select rtrim('  Hello World  ')")
-    
+    testFoldConst("select rtrim('abc ')")
+    testFoldConst("select rtrim('abc,,,', ',')")
+    testFoldConst("select rtrim('Aaabc', 'A')")
+    testFoldConst("select rtrim('Aaabc', 'c')")
+    testFoldConst("select rtrim('abc;;;', ';')")
+    testFoldConst("select rtrim('abc, ', ' ,')")
+    testFoldConst("select rtrim(null)")
+    testFoldConst("select rtrim('abc', null)")
+    testFoldConst("select rtrim(null, null)")
+    testFoldConst("select rtrim('abcxyz', 'xyz')")
+    testFoldConst("select rtrim('abcxyxzy', 'xyz')")
+    testFoldConst("select rtrim('')")
+    testFoldConst("select rtrim('', 'a')")
+    testFoldConst("select rtrim(123)")
+    testFoldConst("select rtrim(0)")
+    testFoldConst("select rtrim(true)")
```

Comment:
add some emoji and minority language cases

Reply:
added